caboose.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	EnvironmentKey = "STRN_ENV_TAG"
+	STRN_ENV_KEY = "STRN_ENV_TAG"
```

Comment:
use `SaturnEnvKey` as this camel-case style is used for all other caps

Reply:
Done.